Jenkinsfile

```diff
@@ -33,29 +33,31 @@ def vagrantfile = '''
 require 'uri'
 
 node_id = URI::encode(ENV['NODE_ID'] || '')
-name = "mojave-xcode11-#{ENV['BRANCH_NAME'] || ''}"
+name = "catalina-xcode11.3-#{ENV['BRANCH_NAME'] || ''}"
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "mojave-xcode11"
+    config.vm.box = "catalina-xcode11.3"
+    config.vm.synced_folder ".", "/vagrant", disabled: true
 
-    config.vm.define "mojave" do |mojave|
-        mojave.vm.hostname = "mojave-xcode11"
-        mojave.ssh.forward_agent = true
+    config.vm.define "catalina" do |catalina|
+        catalina.vm.hostname = "catalina-xcode11.3"
```

Comment:
hosename is same as `config.vm.box` maybe we can say `catalina.vm.hostname = config.vm.box`?

Reply:
That was a mistake, the box name is just catalina. I will update it.